### PR TITLE
Printf

### PR DESCRIPTION
**Pull request by Metacinnabar**
## Checklist

- [x] I have read the [Contributing Guidelines]
- [x] I acknowledge that any submitted code will be licensed under the [ISC License]
- [x] I confirm that submitted code is my own work
- [x] I have tested the code, and confirm that it works

## Enviroment

- Operating System: Linux PopOS
- Node version: N/A (bash)
- NPM version: N/A (bash)

## Description

Switch `echo` to `printf` as `echo` appends a newline that will then be copied with the url.

### CODE REVIEW

File: flameshot_example.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 IMAGEPATH="$HOME/Pictures/" # Where to store screenshots before they're deleted
 IMAGENAME="ass" # Not really important, tells Flameshot what file to send and delete
 KEY="" # Your ass upload token
-DOMAIN="" # Your upload domain
+DOMAIN="" # Your upload domain (without http(s))
```
Comment (tycrek) [inline flameshot_example.sh:6]: Can you please revert this change? Some users do use HTTPS with Flameshot

Comment (tycrek) [reply flameshot_example.sh:6]: Actually hold on I may have misunderstood

Comment (tycrek) [reply flameshot_example.sh:6]: @Metacinnabar Can you actually *change* that comment to be:

```bash
DOMAIN="" # Your upload domain (without http:// or https://)

**Review by tycrek — CHANGES_REQUESTED** [review]
Just need that one line changed and I'll merge

File: flameshot_example.sh (hunk 1 above)
Comment (Metacinnabar) [reply flameshot_example.sh]: Done!

**Review by tycrek — APPROVED** [review]
Looks good, thanks!
